src/Symfony/Component/Config/Resource/FileExistenceResource.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config\Resource;
+
+/**
+ * FileExistenceResource represents a resource stored on the filesystem.
+ * Freshness is only evaluated against resource creation or deletion.
+ *
+ * The resource can be a file or a directory.
+ *
+ * @author Charles-Henri Bruyand <charleshenri.bruyand@gmail.com>
+ */
+class FileExistenceResource implements ResourceInterface, \Serializable
+{
+    /**
+     * @var string|false
+     */
+    private $resource;
+
+    /**
+     * @var boolean
+     */
+    private $exists;
+
+    /**
+     * Constructor.
```

Comment:
No, it's not part of your PR. I just don't want to introduce new comments like this.